Source/VolunteerReporting/Read/CaseReports/CaseReport.cs

```diff
@@ -1,11 +1,13 @@
 using System;
+using Concepts;
 
-namespace Read.CaseReportFeatures
+namespace Read.CaseReports
 {
     public class CaseReport
     {
         public Guid Id { get; set; }
         public Guid DataCollectorId { get; internal set; }
+        public string Origin { get; internal set; }
```

Comment:
I struggled back and forth with this - and I might have gone one too far on this. My thinking was that CaseReporting has little to do with SMS - and a phone number in the middle there felt like bleeding the transport details into it all. I know it doesn't really fix it with the "origin" thing. This particular read model is for a case where we know the data collector, so it shouldn't have it on it. The other one has this. 

One option for this could be to have a metadata object associated instead. Not sure. 
Maybe it is explicit that a Unknown data collector would only be unknown if coming from a text message and there reflect it in naming throughout, even the name of the event and read model - somehow. Maybe thats the metadata for us to identify it.